rpm/lua-rex.spec

```diff
@@ -0,0 +1,257 @@
+Name:           lrexlib
+Version:        2.9.0
+Release:        1%{?dist}
+Summary:        Regular expression handling library for Lua
+
+Group:          Development/Libraries
+License:        MIT
+URL:            http://lrexlib.luaforge.net/
+Source0:        lrexlib-%{version}.tar.gz
+
+%description
+Lrexlib are bindings of six regular expression library APIs (POSIX, PCRE, PCRE2,
+Oniguruma, Tre and GNU) to Lua.
+
+%package pcre
+Summary: Lua binding of PCRE library
+BuildRequires:  pcre-devel
```

Comment:
It linked statically of dynamically? We need to add it to runtime requires in the latter case.

Reply:
Dynamically.
During package install:
```
--> Processing Dependency: libpcre2-8.so.0()(64bit) for package: lrexlib-pcre2-2.9.0-1.el7.centos.x86_64
```
This is confusing, we don't mention it in `Requires`, but PRM still silently 'put' it there and then require during install.
Added pcre, pcre2, etc in `Requires`

Reply:
That is interesting why (according to which rules) rpmbuild adds new Requires to an RPM package. But anyway it is better to have runtime dependencies in Requires in explicit way.

Reply:
I think it works the following way:
Our package require pcre-devel. Package pcre provides pcre-devel in addition to pcre. So RPM just use pcre package.